src/btree_private.h

```diff
@@ -223,11 +223,11 @@ btree_get_leaf_entry(const btree_config *cfg,
     */
    debug_assert(diff_ptr(hdr, &hdr->offsets[hdr->num_entries])
                 <= hdr->offsets[k]);
-   debug_assert(hdr->offsets[k] + sizeof(leaf_entry) <= btree_page_size(cfg));
+   debug_code(size_t bt_page_size = btree_page_size(cfg));
```

Comment:
All the page size functions return uint64.  Let's keep types consistent.

Reply:
Done. Found, and fixed, a few more such instances in other files in this change-set which declared `size_t`. These have also been changed to `uint64`, to be consistent with how they are being used in those files.